sharktank_models/llama3.1/test_llama.py

```diff
@@ -0,0 +1,111 @@
+import iree.compiler
```

Comment:
Please add a license header comment to this file.

Reply:
Done.